doc/ref_impl/cntt-ri/chapters/chapter04.md

```diff
@@ -7,60 +7,68 @@
 * [4.1 Introduction](#4.1)
 * [4.2 Lab Requirement](#4.2)
 * [4.3 Lab Topology](#4.3)
-* [4.4 Lab HW Spec](#4.4)
-* [4.5 Lab Use Guidelines](#4.5)
+* [4.4 Lab Use Guidelines](#4.4)
 
 
 <a name="4.1"></a>
-## 4.1 Introdcution
+## 4.1 Introduction
+This chapter provides the laboratory hardware requirements needed to deploy the reference implementation.  These requirements represent a minimum set of requirements, where different labs may have hardware or capabilities exceeding these minimum specifications.  
 
+The lab provides a set of physical servers, supported by a dedicated network infrastructure.  Users should be provided access to the servers, their operating systems, and out-of-band management (i.e. IPMI, IDRAC, ILO, etc.).  
 
 <a name="4.2"></a>
-## 4.2 Lab Requirement
-
-- Controller Nodes:
-  - 3 x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD.
-    - 3.2TB SSD.
-- Compute Node
-  - 10x
-    - 2x dual-port 25Gbps NIC.
-    - 2.2GHz 24C/48T.
-    - 512GB RAM.
-    - 2TB HDD
-    - Supports 2x Nodes for B/N/C flavors
-- Jump/Baremetal manager
-  - 1x
-    - 2x dual-port 10Gbps NIC.
-    - 2.2GHz 14C/28T.
-    - 256GB RAM.
-    - 10TB HDD
-- Networking
-  - 1x Spine Switch
-    - Total: 32x100G
-  - 2x Leaf Switch
-    - 48 x 25/10G
-    - 6 x 100G
-- 1x 48u Rack
-- Cables and Transcievers
-
-<a name="4.3"></a>
-## 4.3 Lab Topology
+## 4.2 Lab Requirements
 
+Labs are organized into one or more *Pods*, where the *Pod* provides the complete set of resources [servers and network(s)] necessary to support the reference implementation installation and operation.  
 
-<a name="4.4"></a>
-## 4.4 Lab HW Spec
+A CNTT compliant *Pod* shall meet the following requirements.
```

Comment:
We can loosen that up. It should apply to minimum lab requirements. Are you ok if we update this after the go/no-go?

Reply:
I think we should insert some of the caveats from the governance call here; potentially points 3,4,5.  The document shouldn't imply that these specs are yet ready for use.

From the [Governance call minutes on 2019-11-21](https://wiki.lfnetworking.org/pages/viewpage.action?pageId=25363522) 

> AI - Approved by Governance with caveats:
> 1. This requires Technical Steering discussion and approval of specs.
> 2. This is a UNH owned system.
> 3. UNH would need to understand there is risk because RI1 specs not finalized, and may change.
> 4. CNTT not legally bound to specs.
> 5. Stipulate needs to only be used at UNH.